blackberggroup/VA-National-Centers
# Bug/jls 7487958512/rm dup content add ncchi necie

## What type of PR is this? (check all applicable)


## Description NCI2I replace duplicate content with NCCHI and NECIE. Change "Pathfinder" to "PathFinder" in main nav. Correct the path for NECIE in popup on home page map.

## Related Tickets & Documents
- Monday Task https://blackberg-group.monday.com/boards/1931719392/pulses/7487958512

## QA Instructions, Screenshots, Recordings
Ensure content has been corrected: replace duplicate DRHCI and IDEAS with NCCHI and NECIE. Ensure that content breaks down for desktop/tablet/mobile. Ensure links are correct

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _includes/header-sub.html
 - _includes/header.html
 - _includes/home/national-centers.html
 - _site/assets/js/vendors/map-svg/mapsvg.functions.js
 - _site/drhci/index.html
 - _site/ideas/index.html
 - _site/index.html
 - _site/nc4ti/index.html
 - _site/necie/index.html
 - _site/va-ventures/index.html
 - assets/js/vendors/map-svg/mapsvg.functions.js
